platform-sdk/swirlds-platform-core/src/testFixtures/java/com/swirlds/platform/test/fixtures/event/source/AbstractEventSource.java

```diff
@@ -228,10 +229,12 @@ public IndexedEvent generateEvent(
         // work for consensus, and we will have to upgrade our simulation framework prior to converting
         // consensus to use the birth round.
         final IndexedEvent latestSelfEvent = getLatestEvent(random);
-        final long birthRound = Math.max(
-                        otherParentEvent == null ? 0 : otherParentEvent.getGeneration(),
-                        latestSelfEvent == null ? 0 : latestSelfEvent.getGeneration())
+        final long generation = Math.max(
+                        otherParentEvent == null ? (FIRST_GENERATION - 1) : otherParentEvent.getGeneration(),
+                        latestSelfEvent == null ? (FIRST_GENERATION - 1) : latestSelfEvent.getGeneration())
                 + 1;
+        // First generation is 0, but first birth round is 1.
+        final long birthRound = generation + 1;
```

Comment:
Would it make more sense to use `ConsensusConstants.ROUND_FIRST`.   or just a plain `1`.

Reply:
In this case the "+ 1" is my preference. Even though `ROUND_FIRST` is equal to 1, it's not implied by the current documentation that adding the first round to a generation makes it a legal birth round number.